packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -1,8 +1,20 @@
 /*!
- * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
 import {SparkPlugin, Page} from '@ciscospark/spark-core';
+import {
+  WEBEX_EVENT,
+  API_ACTIVITY_VERB,
+  SDK_EVENT_RESOURCE,
+  SDK_EVENT,
+  SDK_EVENT_OWNER,
+  SDK_EVENT_STATUS,
+  hydraTypes,
+  constructHydraId
+} from '@ciscospark/common';
+
+const debug = require('debug')('memberships');
```

Comment:
What does this do? Seems like it should be for dev only?

Reply:
The debug module?  It provides a way for developers to see any debug output from the modules.  This is there for most of the public modules (I followed the debug model that Matt used when implementing the message event)